apps/namadillo/src/atoms/notifications/functions.ts

```diff
@@ -1,21 +1,11 @@
-import { TxProps } from "@namada/types";
-
 export const notificationIdSeparator = ";";
+import { TxProps } from "@namada/types";
 
 export const createNotificationId = (
-  data?: TxProps | TxProps[] | string
+  data?: TxProps["hash"] | TxProps["hash"][]
```

Comment:
Funny, I was thinking that you will allow passing a bunch of different types and handle them all here returning `string[]` instead of preserving the `data.join(";")`